diff --git a/node_modules/@vitalets/google-translate-api/index.d.ts b/node_modules/@vitalets/google-translate-api/index.d.ts
index ca60540..b5847f1 100644
--- a/node_modules/@vitalets/google-translate-api/index.d.ts
+++ b/node_modules/@vitalets/google-translate-api/index.d.ts
@@ -9,6 +9,7 @@ declare namespace googleTranslateApi {
   export interface IOptions {
     from?: string;
     to?: string;
+    textId?: string
   }
 
   export interface ITranslateLanguage {
@@ -24,6 +25,7 @@ declare namespace googleTranslateApi {
 
   export interface ITranslateResponse {
     text: string;
+    textId: string
     pronunciation: string;
     from: {
       language: ITranslateLanguage;
diff --git a/node_modules/@vitalets/google-translate-api/index.js b/node_modules/@vitalets/google-translate-api/index.js
index 0c10f6d..a605de1 100644
--- a/node_modules/@vitalets/google-translate-api/index.js
+++ b/node_modules/@vitalets/google-translate-api/index.js
@@ -63,6 +63,7 @@ function translate(text, opts, gotopts) {
 
             var result = {
                 text: '',
+                textId: opts.textId || Date.now(),
                 pronunciation: '',
                 from: {
                     language: {
