diff --git a/node_modules/react-native-web/dist/index.js b/node_modules/react-native-web/dist/index.js
index f53b5dc..1c625a0 100644
--- a/node_modules/react-native-web/dist/index.js
+++ b/node_modules/react-native-web/dist/index.js
@@ -73,3 +73,4 @@ export { default as DeviceEventEmitter } from './exports/DeviceEventEmitter'; //
 export { default as useColorScheme } from './exports/useColorScheme';
 export { default as useLocaleContext } from './exports/useLocaleContext';
 export { default as useWindowDimensions } from './exports/useWindowDimensions';
+export const ViewPropTypes = { style: ()=> null };
